src/csharp/Grpc.Auth/project.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "0.14.0-anexperiment",
+  "title": "gRPC C# Auth",
+  "authors": [ "Google Inc." ],
+  "copyright": "Copyright 2015, Google Inc.",
+  "packOptions": {
+    "summary": "Auth library for C# implementation of gRPC - an RPC library and framework",
+    "description": "Auth library for C# implementation of gRPC - an RPC library and framework. See project site for more info.",
+    "owners": [ "grpc-packages" ],
+    "licenseUrl": "https://github.com/grpc/grpc/blob/master/LICENSE",
+    "projectUrl": "https://github.com/grpc/grpc",
+    "requireLicenseAcceptance": false,
+    "tags": [ "gRPC RPC Protocol HTTP/2 Auth OAuth2" ],
+  },
+  "dependencies": {
+    "Grpc.Core": "0.14.0-anexperiment",
+    "Google.Apis.Auth": "1.11.1"
+  },
+  "frameworks": {
+    "net45": { },
+    "netstandard1.5": {
+      "imports": [
```

Comment:
Yes, this is because Google.Api.Auth is not supporting coreclr yet. Having this allows me to add a dependency and make things compile "as if" it was portable yet.

I would have added an explanatory comment, but project.json files don't support comments (which is actually pretty painful).